packages/components/src/components/icon/common/ic-success-reset-trading-password.svg

```diff
@@ -1 +1 @@
-<svg width="130" height="133" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M12.045 33h75.91A12.045 12.045 0 01100 45.045v75.91A12.044 12.044 0 0187.955 133h-75.91A12.044 12.044 0 010 120.955v-75.91A12.045 12.045 0 0112.045 33zm21.2 24.69a17.915 17.915 0 00-1.146 7.078v4.667h8.07v-4.547a9.846 9.846 0 1119.662 0v4.547h8.07v-4.668a17.915 17.915 0 00-34.656-7.078zm43.19 59.596a6.295 6.295 0 001.855-4.431v-32.58a6.293 6.293 0 00-6.294-6.293H28.004a6.293 6.293 0 00-6.294 6.293v32.58a6.295 6.295 0 006.294 6.264h43.992c1.664 0 3.26-.659 4.44-1.833zm-30.31-30.078a6.293 6.293 0 013.89-1.484v.06a6.293 6.293 0 012.349 12.045v9.877h-4.728v-9.937a6.293 6.293 0 01-1.512-10.561z" fill="#85ACB0"/><g style="mix-blend-mode:multiply"><path d="M95.833 66.76c18.41 0 33.334-14.923 33.334-33.333 0-18.41-14.924-33.333-33.334-33.333C77.423.094 62.5 15.018 62.5 33.427c0 18.41 14.924 33.333 33.333 33.333z" fill="#FF444F"/></g><path d="M91.11 46.621L79.167 34.677l5-5 6.945 6.944L107.5 20.232l5 5-21.39 21.39z" fill="#fff"/></svg>
\ No newline at end of file
+<svg width="130" height="133" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M12.045 33h75.91A12.045 12.045 0 01100 45.045v75.91A12.044 12.044 0 0187.955 133h-75.91A12.044 12.044 0 010 120.955v-75.91A12.045 12.045 0 0112.045 33zm21.2 24.69a17.915 17.915 0 00-1.146 7.078v4.667h8.07v-4.547a9.846 9.846 0 1119.662 0v4.547h8.07v-4.668a17.915 17.915 0 00-34.656-7.078zm43.19 59.596a6.295 6.295 0 001.855-4.431v-32.58a6.293 6.293 0 00-6.294-6.293H28.004a6.293 6.293 0 00-6.294 6.293v32.58a6.295 6.295 0 006.294 6.264h43.992c1.664 0 3.26-.659 4.44-1.833zm-30.31-30.078a6.293 6.293 0 013.89-1.484v.06a6.293 6.293 0 012.349 12.045v9.877h-4.728v-9.937a6.293 6.293 0 01-1.512-10.561z" fill="#85ACB0"/><g><path style="mix-blend-mode:multiply" d="M95.833 66.76c18.41 0 33.334-14.923 33.334-33.333 0-18.41-14.924-33.333-33.334-33.333C77.423.094 62.5 15.018 62.5 33.427c0 18.41 14.924 33.333 33.333 33.333z" fill="#FF444F"/></g><path d="M91.11 46.621L79.167 34.677l5-5 6.945 6.944L107.5 20.232l5 5-21.39 21.39z" fill="#fff"/></svg>
```

Comment:
@nikita-deriv What's changed in this SVG file? Github shows they are like each other! 
![image](https://user-images.githubusercontent.com/55829032/123032095-a34dc600-d3fa-11eb-97a3-1f711b508260.png)

Reply:
`style="mix-blend-mode:multiply"` was moved from `<g>` tag to `<path>` tag.

Reply:
@nikita-deriv I saw that but it seems that change doesn't affect its appearance. that red circle is still transparent. is that OK?

Reply:
No, this is not ok. And this is strange, because I see correct view:
<img width="524" alt="Screenshot 2021-06-23 at 10 50 10" src="https://user-images.githubusercontent.com/84083878/123057787-d56a2280-d410-11eb-9dd3-d8afcd168dc5.png">

Reply:
I think you don't need to keep the `style="mix-blend-mode:multiply"` part. simply, just remove that.

Reply:
In my case removing this style property causing transparent view.

Reply:
I have tested it and it was  ok for me after removing that `style` part